dashboard/modules/job/job_agent.py

```diff
@@ -190,6 +190,8 @@ async def tail_job_logs(self, req: Request) -> Response:
         await ws.prepare(req)
 
         async for lines in self._job_manager.tail_job_logs(job.submission_id):
+            # NOTE: Sending chunk over the web-socket is an async operation,
+            #       allowing sync tailing iteration to yield the event-loop
```

Comment:
What does this mean? `tail_job_logs` is an async iterator. so don't know what you mean by "sync tailing iteration"